src/raygun.rum.js

```diff
@@ -489,7 +490,11 @@ var raygunRumFactory = function(window, $, Raygun) {
                 collection.push({
                   url: response.baseUrl,
                   statusCode: response.status,
-                  timing: { du: response.duration, t: Timings.XHR },
+                  timing: { 
+                    du: maxFiveMinutes(response.duration).toFixed(2), 
```

Comment:
`toFixed` returns a string, will this be an issue?

Reply:
It won't be a issue as the API converts it to the correct data type during ingestion. (Although we should be more consistent with how we return data, currently the duration are strings instead of numbers during the regular ingestion)